src/core/sys/linux/sys/socket.d

```diff
@@ -0,0 +1,17 @@
+/**
+ * D header file for GNU/Linux.
+ *
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Paul O'Neil
+ */
+module core.sys.linux.sys.socket;
+
+version(linux):
+extern(C):
+@nogc:
+nothrow:
+
+enum {
```

Comment:
It should publicly `import core.sys.posix.sys.socket;` so it functions as an overlay.